tests/xtro-sharpie/api-annotations-dotnet/common-WebKit.ignore

```diff
@@ -0,0 +1,5 @@
+# xcode 15
+# requires OS_nw_proxy_config type which has not been bound yet, part of Xcode 15 Network binding
+!missing-selector! WKWebsiteDataStore::proxyConfigurations not bound
+!missing-selector! WKWebsiteDataStore::setProxyConfigurations: not bound
```

Comment:
@haritha-mohan  can you please create an issue to track this, I'll take care of it once I do the network framework.

Reply:
👍 https://github.com/xamarin/xamarin-macios/issues/19259